### DISCUSSION TITLE
new clients layout

### DISCUSSION THREAD

**Discussion by spiralshapeturtle** [AdguardTeam/AdGuardHome#3233] [Ideas]
Hi

the query log is improved great the last updates. I'm using the client page a lot, and that's still the old fashioned page. No reliable search, default layout 10 clients. The best thing to manage clients seems to be ctrl-f, that could be done better. An option to see which checkboxes are checked when "hovering" over a clients name would be great also.

May I ask to improve the client page?

**Comment by ameshkov** [comment]
Well, we definitely will improve it:)

**Reply by spiralshapeturtle** [reply]
Hi

what also will improve the networks with redundant Aduard servers: config export and import. Just to keep them in sync with the same client settings and blocklists.
